packages/dapp/src/config.ts

```diff
@@ -53,34 +53,34 @@ if (
 
 const allowedNetworks: NetworkInfo[] = [
   {
-    name: 'Hardhat Testnet',
-    chainId: 1337,
-    address: '0x5FbDB2315678afecb367f032d93F642f64180aa3',
+    name: "Hardhat Testnet",
+    chainId: 31337,
```

Comment:
still not fixed yet? please contact Valera. You both have to fix this issue one time and forever